storage/tokudb/ha_tokudb.cc

```diff
@@ -1730,13 +1730,16 @@ int ha_tokudb::initialize_share(const char* name, int mode) {
     if (!hidden_primary_key) {
         //
         // We need to set the ref_length to start at 5, to account for
-        // the "infinity byte" in keys, and for placing the DBT size in the first four bytes
+        // the "infinity byte" in keys, and for placing the DBT size in the
+        // first four bytes
         //
         ref_length = sizeof(uint32_t) + sizeof(uchar);
-        KEY_PART_INFO *key_part = table->key_info[primary_key].key_part;
-        KEY_PART_INFO *end = key_part + get_key_parts(&table->key_info[primary_key]);
+        KEY_PART_INFO* key_part = table->key_info[primary_key].key_part;
+        KEY_PART_INFO* end = key_part + get_key_parts(&table->key_info[primary_key]);
```

Comment:
Yeah, we are, this just hasn't been merged/rebased with the other branch yet that needs to use key->user_defined_key_parts instead of the depreated Toku get_key_parts function.

I'm merging the other branch with fixups based on comments and will rebase this one before merging it which will eliminate the get_key_parts call.